packages/manager/modules/exchange/src/domain/spf-autoconfig/domain-spf-autoconfig.controller.js

```diff
@@ -0,0 +1,111 @@
+export default class ExchangeDomainSpfAutoconfigCtrl {
+  /* @ngInject */
+  constructor(
+    $scope,
+    wucExchange,
+    ExchangeDomains,
+    navigation,
+    messaging,
+    $translate,
+    exchangeStates,
+  ) {
+    this.services = {
+      $scope,
+      wucExchange,
+      ExchangeDomains,
+      navigation,
+      messaging,
+      $translate,
+      exchangeStates,
+    };
```

Comment:
**nitpick:** Unnecessary wrapper object.
```suggestion
      this.$scope = $scope;
      this.wucExchange = wucExchange;
      .
      .
      .
      this.exchangeStates = exchangeStates;
```

Reply:
It's same as packages/manager/modules/exchange/src/domain/srv-autoconfig/domain-srv-autoconfig.controller.js and packages/manager/modules/exchange/src/domain/mx-autoconfig/domain-mx-autoconfig.controller.js already on prod.
To keep simple code and homogeneity, I think it's better to keep it like that until a PR to improve globally code will be made.